### PR TITLE
update fi_usnic(7) man page for the 1.1.1 release

### PR DESCRIPTION
**Pull request by goodell**
@bturrubiates please review

### CODE REVIEW

**Comment by goodell** [comment]
I'll file a PR to bring these and #1310's changes over to master in a few minutes.

File: man/fi_usnic.7.md [hunk 1]
```diff
@@ -20,51 +20,51 @@ low latency and other offload capabilities on Ethernet networks.
...
-    information and functionality beyond the standard libfabric
-    interface.  See the "USNIC EXTENSIONS" section, below.
+    - *FI_EP_DGRAM* endpoints are the best supported method of utilizing
+      the usNIC interface.  Specifically, the *FI_EP_DGRAM* endpoint
+      type has been extensively tested as the underlying layer for Open
+      MPI's *usnic* BTL.
+    - *FI_EP_MSG* and *FI_EP_RDM* endpoints are implemented, but are
+      only lightly tested.  It is likely that there are still some bugs
+      in these endpoint types.  RMA is not yet supported.
+    - *FI_EP_RDM* support is not 100% robust and occasionally leads to hangs in
```
Comment (bturrubiates) [inline man/fi_usnic.7.md]: This can be removed, I think the `fi_rdm_pingpong` is a solved problem.

File: man/fi_usnic.7.md [hunk 2]
```diff
@@ -20,51 +20,51 @@ low latency and other offload capabilities on Ethernet networks.
...
+    - [`fi_provider`(7)](fi_provider.7.html) lists requirements for all
+      providers.  The following limitations exist in the *usnic*
+      provider:
+        * multicast operations are not supported on *FI_EP_DGRAM* and
+          *FI_EP_RDM* endpoints.
+        * *FI_EP_MSG* endpoints only support connect, accept, getname, and
+          shutdown CM operations.
+        * Passive endpoints only support listen, setname, and getname CM
+          operations.
+        * *FI_EP_DGRAM* endpoints support `fi_sendmsg()` and
```
Comment (bturrubiates) [inline man/fi_usnic.7.md]: `fi_sendmsg()` supports `FI_INJECT` and `FI_COMPLETION`.

File: man/fi_usnic.7.md [hunk 3]
```diff
@@ -20,51 +20,51 @@ low latency and other offload capabilities on Ethernet networks.
...
+          `fi_recvmsg()`, but all flags are ignored.
```
Comment (bturrubiates) [inline man/fi_usnic.7.md]: `fi_recvmsg()` supports `FI_MORE`.

**Comment by bturrubiates** [comment]
Other than the other comments, looks good :+1:

**Comment by goodell** [comment]
fixed, pulling

**Pull request merged**